docs/sql-ref-syntax-ddl-alter-table.md

```diff
@@ -75,6 +75,54 @@ ALTER TABLE table_identifier ADD COLUMNS ( col_spec [ , ... ] )
 
     Specifies the columns to be added.
 
+### DROP COLUMNS
+
+`ALTER TABLE DROP COLUMNS` statement drops mentioned columns of an existing table.
+Note that this statement is only supported with v2 tables.
```

Comment:
ALTER COLUMN can't change column type but we don't mention it in the doc. I think it's OK to not mention the limitation here as well.